android/anc/src/main/java/org/smartregister/fhircore/anc/ui/otp/OtpLoginScreen.kt

```diff
@@ -0,0 +1,265 @@
+import androidx.compose.material.Surface
+import androidx.compose.material.Text
+import androidx.compose.material.TopAppBar
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.ArrowBack
+import androidx.compose.material.icons.outlined.MoreVert
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.livedata.observeAsState
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.ExperimentalComposeUiApi
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.res.colorResource
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.text.style.TextDecoration
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import org.smartregister.fhircore.anc.R
+import org.smartregister.fhircore.anc.ui.family.details.TOOLBAR_MENU
+import org.smartregister.fhircore.anc.ui.family.details.TOOLBAR_MENU_BUTTON
+import org.smartregister.fhircore.anc.ui.family.details.TOOLBAR_TITLE
+import org.smartregister.fhircore.engine.ui.components.PinView
+import org.smartregister.fhircore.engine.ui.login.APP_LOGO_TAG
+import org.smartregister.fhircore.engine.ui.theme.LoginButtonColor
+import org.smartregister.fhircore.engine.ui.theme.LoginDarkColor
+import org.smartregister.fhircore.engine.util.annotation.ExcludeFromJacocoGeneratedReport
+
+const val TOOLBAR_MENU_ICON = "toolbarIcon"
+const val TOOLBAR_MENU_LOGIN = "toolbarMenuLogin"
+const val FORGOT_PIN = "forgot_pin"
+
+@Composable
+fun OtpLoginScreen(viewModel: OtpViewModel) {
+
+  val showError by viewModel.showError.observeAsState(initial = false)
+
+  OtpLoginPage(
+    onPinChanged = viewModel::onPinChanged,
+    showError = showError,
+    onMenuLoginClicked = { viewModel.onMenuLoginClicked() },
+    forgotPin = viewModel::forgotPin,
+  )
+}
+
+@OptIn(ExperimentalComposeUiApi::class)
+@Composable
+fun OtpLoginPage(
+  modifier: Modifier = Modifier,
+  onPinChanged: (String) -> Unit,
+  showError: Boolean = false,
+  onMenuLoginClicked: () -> Unit,
+  forgotPin: () -> Unit
+) {
+
+  var showMenu by remember { mutableStateOf(false) }
+  var showForgotPinDialog by remember { mutableStateOf(false) }
+
+  Surface(color = colorResource(id = R.color.white_slightly_opaque)) {
+    TopAppBar(
+      title = { Text(text = "", Modifier.testTag(TOOLBAR_TITLE)) },
+      navigationIcon = {
+        IconButton(onClick = {}) {
+          Icon(
+            Icons.Filled.ArrowBack,
+            contentDescription = "Back arrow",
```

Comment:
Move this to a stings file if shown to the users. We need to account for MLS (Multi-language support)

Reply:
its image view content description will not reflecting in app so doesn't required MLS,